server/api/releases/[id]/complete.get.ts

```
@@ -0,0 +1,139 @@
+				`)
+				.eq('release_id', releaseId)
+			finalReleaseMusics = altMusics
+		}
+
+		// Récupérer des releases suggérées (même artiste, excluant le release actuel)
+		const artistIds = finalReleaseArtists?.map((junction: any) => junction.artist.id) || []
+		let suggestedReleases = []
+
+		if (artistIds.length > 0) {
+			// Approche simplifiée : récupérer d'abord les IDs des releases suggérées
+			let { data: suggestedIds } = await supabase
```

ESLint

[error] 'suggestedIds' is never reassigned. Use 'const' instead.

(prefer-const)